### DISCUSSION TITLE
Search thru files for specific word

### DISCUSSION THREAD

**Discussion by Pok4** [WinMerge/winmerge#1241] [Q&A] [answered]
Can i loop thru all files for specific word like font "Monofont" (i want to search in all files for this word)

Maybe the answer is funny, but i need this one to check my files because i have assets/font/Monofont.ttf and i don't know from where the files using this font... The system is big...

**Comment by sdottaka** [comment, marked as answer]
Unfortunately, WinMerge does not have the ability to enumerate text files containing specified words like the grep tool.

A diff tool called Diffinity seems to have that functionality.

https://truehumandesign.se/s_diffinity.php

**Comment by Pok4** [comment]
Thank you! I search in google and found https://github.com/stefankueng/grepWin (it works for me) :)

**Comment by Gitoffthelawn** [comment]
Just FYI, Notepad++ (https://github.com/notepad-plus-plus/notepad-plus-plus) can do this too.
